ui/src/app/components/status-alert-banner.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+
+import {Button} from 'app/components/buttons';
+import {FlexColumn, FlexRow} from 'app/components/flex';
+import {ClrIcon} from 'app/components/icons';
+import {statusAlertApi} from 'app/services/swagger-fetch-clients';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import {cookiesEnabled} from 'app/utils';
+
+const styles = {
+  alertBanner: {
+    backgroundColor: colorWithWhiteness(colors.highlight, .5),
+    color: colors.primary,
+    marginRight: '1rem',
+    marginTop: '1rem',
+    padding: '0.5rem',
+    width: '300px',
+    borderRadius: '0.5em',
+    position: 'absolute',
+    top: '0',
+    right: '0',
+    zIndex: '1',
+  }
+};
+
+const cookieKey = 'status-alert-banner-dismissed';
+
+export interface State {
+  statusAlertId: number;
+  title: string;
+  message: string;
+  link: string;
+  isVisible: boolean;
+}
+
+export class StatusAlertBanner extends React.Component<{}, State> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      statusAlertId: 0,
+      title: '',
+      message: '',
+      link: '',
+      isVisible: false
+    };
+  }
+
+  async componentDidMount() {
+    const statusAlerts = await statusAlertApi().getStatusAlert();
+    if (statusAlerts.length > 0) {
+      // There should be either one or zero alerts.
```

Comment:
The API suggests that there could be N status alerts. TOL: If we are expecting only one or zero alerts, should it be returning a list? (I'd expect a nullable object)

TOL: I think it is fine to have it return as a list, I'd just remove this comment? Because I could see us future proofing against multiple alerts. (I think Lou has comps of multiple with tabs in the tooltip for future work)

Reply:
I'll change it to returning an `Option<Alert>`, I think I was thinking too pythonic or something.